tests/components/StaticFilters.test.tsx

```diff
@@ -158,12 +156,16 @@ describe('Static Filters', () => {
     const actions = spyOnActions();
     render(<StaticFilters {...staticFiltersProps} />);
 
-    const martyCheckbox: HTMLInputElement = screen.getByLabelText('MARTY!');
+    const martyFilter = staticFiltersProps.filterOptions[0];
+    const martyCheckbox: HTMLInputElement = screen.getByLabelText(martyFilter.label);
+    expect(martyCheckbox.checked).toBeFalsy();
+
     userEvent.click(martyCheckbox);
-    expect(actions.executeVerticalQuery).toBeCalled();
+    expectFilterOptionSet(actions, staticFiltersProps.fieldId, martyFilter, true);
```

Comment:
I think we already check that setFilterOption is called correctly when clicking a checkbox option in other tests above, so not sure it's needed here as well